fnirsi_logger.py

```diff
@@ -24,11 +24,13 @@
 
 def main():
     # find our device
+    is_fnb58 = False
     dev = usb.core.find(idVendor=VID, idProduct=PID_FNB48)
     if dev is None:
         dev = usb.core.find(idVendor=VID, idProduct=PID_C1)
     if dev is None:
         dev = usb.core.find(idVendor=VID_FNB58, idProduct=PID_FNB58)
+        is_fnb58 = True
```

Comment:
Lets make this:

```python3
if dev is not None:
  is_fnb58 = True
```

And introduce similar one for fnb48.

For now these bools are fine, but using enum (i.e. `class DeviceModel(enum.Enum)` , or `DeviceQuirks`) is also an option.

Reply:
I suggest to use some abstraction such as my [`DeviceInfo`](https://github.com/didim99/witrn-ui/blob/84cfc265df96261ff1ff0b6b52883d1398291538/driver/protocol.py#L7) if we need it. In future I plan to merge all open source solutions related to USB-meters and other similar equipment (such as loads and programmable PSUs) in one universal data logging/control tool and these abstractions may be useful for common extensible driver architecture.

Reply:
Yes. Now with `data[0] == 0x04` we can skip this.

Resolving.